samplecode/mio/server/enclave/src/lib.rs

```diff
@@ -208,7 +211,7 @@ impl Connection {
             self.do_tls_write();
         }
 
-        if self.closing && !self.tls_session.wants_write() {
+        if self.closing {
```

Comment:
a bug in rustls https://github.com/ctz/rustls/issues/285
Fixed at https://github.com/ctz/rustls/commit/a93ee1abd2ab19ebe4bf9d684d56637ee54a6074